mdn/jinja2/mdn/feature_page_detail.html

```diff
@@ -94,21 +98,22 @@
      <a href="{{ object.feature.mdn_uri['en'] + '$edit' }}" target="_blank">Fix the MDN page</a>
      {% if can_refresh_mdn_import(request.user) %}
      and use Reset to download the new page and reparse.
+     {% else %}
+     and Sign In as an editor to download the new page and reparse.
      {% endif %}
   {% elif object.converted_compat == object.CONVERTED_NO_DATA %}
-    This page has no specification or compatibility data. No further action required.
+    This page has no specification or compatibility data. Go look at a more interesting page.
```

Comment:
Sounds good.  I'm going to redirect them to [pages with errors](https://browsercompat.herokuapp.com/importer/?status=8).